washboard-ui/package.json

```diff
@@ -24,6 +24,7 @@
     "@wasmcloud/prettier-config": "workspace:*",
     "eslint": "^8.57.0",
     "eslint-config-turbo": "^1.13.3",
+    "pino": "^9.0.0",
```

Comment:
Move this one into `washboard-ui/packages/washboard-ui/package.json` and then run `yarn` so that the lockfile is updated and then commit that also.

Reply:
That should resolve the build issue 👍🏻

Reply:
Ah thank you -- just fixed this!